src/main/java/ui/components/issuepicker/IssuePickerDialog.java

```diff
@@ -31,6 +36,11 @@
  */
 public class IssuePickerDialog extends Dialog<String> {
 
+    private static final Background FOCUSED_BACKGROUND = new Background(
+            new BackgroundFill(Color.CORNFLOWERBLUE, CornerRadii.EMPTY, Insets.EMPTY));
+    private static final Background DEFAULT_BACKGROUND = new Background(
+            new BackgroundFill(Color.WHITE, CornerRadii.EMPTY, Insets.EMPTY));
```

Comment:
Should the common part of a group of constants be the prefix? i.e. `BACKGROUND_...` instead of `..._BACKGROUND`.